src/main/java/seedu/address/model/booking/NameContainsKeywordsPredicate.java

```diff
@@ -6,7 +6,8 @@
 import seedu.address.commons.util.StringUtil;
 
 /**
- * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ * Tests that a Booking's {@code TenantName} matches any of the keywords given.
```

Comment:
Btw do we need this file? We dont have a find booking feature right

Reply:
Yup deleted this and it's corresponding test file